### ISSUE TITLE
`--disable-verify` should be the default

### ISSUE THREAD

**Issue by SixthDot** [ldc-developers/ldc#4284]
LLVM IR verification is slow.

**Comment by JohanEngelen** [comment]
Indeed, it speeds up the build at Weka by about 2%. (total, including SemA and linking)
What's the downside? Silent miscompilation, or (better) crash?

**Comment by SixthDot** [comment]
crash durting object emission highly expected.

**Comment by FeepingCreature** [comment]
Jenkins adds new builds at the top. Bottom row is without `-disable-verify`, top row is with `-disable-verify`.
